### DISCUSSION TITLE
[Question]

### DISCUSSION THREAD

**Discussion by rdyfordeparture** [kubilus1/autoortho#361] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: Windows 11
- XPlane Version: XP 11 11.55
- AutoOrtho Version: 0.6.0
- Filesystem type: NTFS

### question

Dear developers and community,

First of all, I would like to thank the developers for creating, maintaining and updating such a lovely product for X-Plane simulators.
Secondly, While using the add-on, I have encountered a problem with cache cleaning. For instance, I have got approx. 12 GB of cache on my C disk folder located in C:/user/.autoortho-data/cache and I`d like to clean it via the exe. I then move the slider to 12 GB and hit clean cache button but I get these messages thereafter:

INFO:config_ui:Cleaning up cache_dir C:\Users\rdyfordeparture\.autoortho-data\cache.  Please wait ...
INFO:config_ui:Cache has 953703 files.  Total size approx 12848 MB.
INFO:config_ui:Found 0 empty files to cleanup.
INFO:config_ui:Cache within size limits.

And the cache folder remains full. Am I doing something wrong or I just don`t fully understand how does this cleaning feature work? Thanks in advance for any replies! :)

**Comment by kubilus1** [comment]
You are setting just a value to prune the cache to where the least recently used things will get cleaned up if over that.  Maybe I don't understand the expectations.

**Comment by Jeff2122** [comment]
I normally set my cache at 50GB. but I wanted to see this work.  It is working as expected I believe. 

My cache had 18GB and set it for 15GB. Got this:
INFO:config_ui:Cleaning up cache_dir F:/cache.  Please wait ...
INFO:config_ui:Cache has 1420800 files.  Total size approx 19093 MB.
INFO:config_ui:Found 0 empty files to cleanup.
INFO:config_ui:Over cache size limit, will remove 277830 files.

I turned around twice in my chair, then ran it again:
INFO:config_ui:Cleaning up cache_dir F:/cache.  Please wait ...
INFO:config_ui:Cache has 1142970 files.  Total size approx 14765 MB.
INFO:config_ui:Found 0 empty files to cleanup.
INFO:config_ui:Cache within size limits.

I had a batch file that completely flushed the cache prior to this feature, this is much better.
